.github/workflows/stellar-dominion-service-build-and-push.yml

```diff
@@ -1,23 +1,19 @@
-name: Build and push stellar-dominion-service docker image
+name: stellar-dominion-service
 
 on:
+  workflow_call:
```

Comment:
The services' workflows can be triggered by a call from another workflow now (in which case they need access to the secret to login into dockerhub).

There's still one case where they can be triggered directly: if the `Dockerfile` associated to the service is changed. In this case we don't have a code change and so it's (probably) safe to assume that we don't need to re-run them.

This will in turn trigger the deployment of the corresponding service (as before).